### ISSUE TITLE
Some Buttons in the Editor Menu bar are not displayed correctly with German Language Pack

### ISSUE THREAD

**Issue by phatair** [osTicket/osTicket#4896]
We are using the following system:
osTicket-Version | v1.11 (d4e240b)
Server-Software | Apache/2.4.18 (Ubuntu)
MySQL-Version | 5.7.25
PHP-Version | 7.0.33-0ubuntu0.16.04.2

Installed Language Pack (it looks like this is the newest version on osticket.com)
Deutsch — de_DE   
de — include/i18n/de.phar 
Version: 154948, for version v1.10 
Created: Wed, 06 Feb 19 21:18:23 +0000

If we use the editor to edit text the options "formatting, table and alignment" is not translated in german. The fields are blank.
You can see this on this screenshot. If we change back to englisch language, the text is correct displayed.

![osticket-editor-german](https://user-images.githubusercontent.com/28873754/57907418-ac642480-787d-11e9-824f-66346a34237b.png)

How can we get the newest language file or how can we edit the existing one to display some text and not a blank field?
Thanks.

**Comment by JediKev** [comment]
@phatair 

I cannot reproduce this issue with the latest stable release of v1.12 and PHP 7.3. We haven't added redactor translations so I'm not sure why yours is being translated. Please download the official language pack from http://osticket.com/download and re-install. 

Cheers.

**Comment by phatair** [comment]
@JediKev 

Thanks but i use the current version from the osticket website.
I downloaded the de.phar file again and copied to the /i18n folder. In this folder are the following files:
de.phar
langs.php
README.md

And the Folder en_US

Is it necessary to copy the en.phar file in this folder, too?
I still have the problems with the empty Buttons.

**Comment by JediKev** [comment]
@phatair 

> Is it necessary to copy the en.phar file in this folder, too?

No, all you have to have is the `en_US/` folder. I installed the German language pack on v1.12 and I do not see the same issue at all...did you customize the code to allow Redactor translations? Maybe a browser plugin?

Cheers.

**Comment by phatair** [comment]
@JediKev 

Hmm...strange. Thats what i have in my i18n folder and i'm using the latest de.phar file from osticket.com.
I dont have any browser plugins installed - i tested it it with IE11 and Chrome.
Maybe i have to upgrade from 1.11 to 1.12 and check if the problem still exists.
